doc/develop/west/build-flash-debug.rst

```diff
@@ -393,12 +393,19 @@ You can :ref:`configure <west-config-cmd>` ``west build`` using these options.
    * - ``build.dir-fmt``
      - String, default ``build``. The build folder format string, used by
        west whenever it needs to create or locate a build folder. The currently
-       available arguments are:
+       supported substitution arguments are:
 
+         - ``west_topdir``: The absolute path to the west workspace, as
+           returned by the ``west_topdir`` command
          - ``board``: The board name
-         - ``source_dir``: The relative path from the current working directory
-           to the source directory. If the current working directory is inside
-           the source directory this will be set to an empty string.
+         - ``source_dir``: Path to the CMake source directory, relative to the
+           current working directory. If the current working directory is
+           inside the source directory, this is an empty string. If no source
+           directory is specified, it defaults to current working directory.
+         - ``source_dir_ws``: Path to the CMake source directory relative to
```

Comment:
`ws` is not a common abreviation, and as this is not something users has to type daily then I would prefer to have it spelled out, like this:
```suggestion
         - ``source_dir_workspace``: Path to the source directory relative to
```